### DISCUSSION TITLE
Web Home page images load very slowly, it takes more than 3 minutes to fully load the home page images

### DISCUSSION THREAD

**Discussion by nodis** [immich-app/immich#10606] [Q&A]
### The bug

Home page images load very slowly, it takes more than 3 minutes to fully load the home page images

CPU：i5 10400
RAM：16GB

### The OS that Immich Server is running on

Synology DSM 7.2

### Version of Immich Server

1.106.4

### Version of Immich Mobile App

1.106.4

### Platform with the issue

- [ ] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
ok
```

### Your .env content

```Shell
ok
```

### Reproduction steps

```bash
1.
2.
3.
...
```

**Comment by bo0tzz** [comment]
What's the CPU and RAM usage on the system, are any of the background jobs running, what is your network connection to the server like, etc?

**Comment by nodis** [comment]
> What's the CPU and RAM usage on the system, are any of the background jobs running, what is your network connection to the server like, etc?

@bo0tzz 
CPU and RAM usage does not exceed 40%;
Tasks running in the background: DUPLICATE DETECTION

**Comment by mmomjian** [comment]
Please wait for the task to finish. That is a very resource heavy task. Did that help once it was done?

**Comment by zackpollard** [comment]
Please let me know if this is still an issue for you and I can help you when I am online tomorrow 😊
